rocksdb_replicator/replicated_db.cpp

```diff
@@ -386,7 +383,7 @@ void RocksDBReplicator::ReplicatedDB::handleReplicateRequest(
 
   // Inverse of predicate below: if requested sequence number is HIGHER than latest sequence number on leader, emit a stat)
   auto leaderSeqNum = db->db_wrapper_->LatestSequenceNumber();
-  if (FLAGS_emit_stat_for_leader_behind && leaderSeqNum < seq_no) {
```

Comment:
should we completely remove the gflag? Is it a good practice to remove a gflag define?

Reply:
the GFLAG is only necessary when we want to flexibly control some behavior without changing the code/binary. In this case, we are using it to control a metrics emission, which I think is an overkill because:
- This is a rare case which shouldn't probably happen in production, so we need this metrics by default to catch the issue, in case it ever happens. 
- The overhead of enabling this metrics is cheap

Given the above reasons, if we keep the GFLAG, it should default to `true` instead.  Then additionally, if it should always be true, that indicates the GFLAG isn't necessary at all. Removing unnecessary configs makes code less complicated and more maintainable.

Reply:
There might be some environments using this gflag. The process doesn't start up if there are unrecognized arguments, so would be good to keep an eye out during deployments or do a sweep of the existing configurations.

Reply:
yep, I did a check of all rockstore & rockstorewidecolumn clusters/stages and don't see any usage of that, but will double check.

Reply:
https://phabricator.pinadmin.com/P11969